chain-signatures/contract/src/lib.rs

```diff
@@ -130,11 +187,14 @@ impl MpcContract {
         }
     }
 
-    fn remove_request(&mut self, request: SignatureRequest) {
+    fn remove_request(&mut self, request: SignatureRequest) -> Result<(), MpcContractError> {
```

Comment:
Is it possible to add a similar return Error type for `sign()`? This is the main function and it is a shame that it does not have strongly typed errors.

Reply:
no I tried, it doesn't work. It will return () instead of the actual yield/resume result. Let me ask Saketh if it's possible.